getstarted.md

```diff
@@ -22,6 +22,70 @@ with Hadoop, as well as extensions to that foundation that make for totally
 awesome demos. You'll also find a few videos related to our work and of course,
 how to contact us!
 
+First, though, let's just take a look at how straightforward it is to get started with very complex software. You can do it during a lunch break:
```

Comment:
I could agree with your approach: Simplify the landing page.

However, this patch does not remove any of the complexity of the current structure. It just places a zeppelin specific tutorial in our landing page. I cannot say I am exited :) Anyway, lets get the opinion of the rest of the team. I might be wrong, my aesthetic was never my strong point.       


I would agree with your approach if that patc